controllers/storagecluster/prometheus/localcephrules.yaml

```diff
@@ -187,6 +187,18 @@ spec:
       for: 1m
       labels:
         severity: critical
+    - alert: CephOSDDown
```

Comment:
An OSD being down isn't really a health warning, if Ceph has since repaired the PGs. This alert would be better if it were only raised if PGs were not clean. But perhaps that's better as a follow-up PR with more time to review the behavior.

Reply:
Agree.